packages/Icon/src/Add.js

```diff
@@ -14,10 +14,10 @@ const SvgAdd = ({ title, ...props }) => (
     focusable={false}
     {...props}
   >
-    <title>{title}</title>
+    {title ? <title>{title}</title> : null}
```

Comment:
Why'd all these Icons get updated in this PR?

Reply:
I'm no sure. I ran `npx lerna bootstrap` and it generated these changes in the js files. Even, so the changes are not for the worse as far as i can tell.

Reply:
I bet there was an update to `svgr`.